### ISSUE TITLE
ValueError: Can't set PROFRQ.dtprofup to 1990-01-01: '1990-01-01' is type '<type 'datetime.date'>'; can't convert to datetime

### ISSUE THREAD

**Issue by fergbrain** [csingley/ofxtools#39]
When running the latest development build and using the profile (-p, --profile) flag the follow error occurs:
```
Traceback (most recent call last):
  File "/usr/local/bin/ofxget", line 11, in <module>
    sys.exit(main())
  File "/Users/fergbrain/Library/Python/2.7/lib/python/site-packages/ofxtools-0.5.1-py2.7.egg/ofxtools/Client.py", line 474, in main
    do_profile(args)
  File "/Users/fergbrain/Library/Python/2.7/lib/python/site-packages/ofxtools-0.5.1-py2.7.egg/ofxtools/Client.py", line 347, in do_profile
    response = client.request_profile(dryrun=args.dryrun)
  File "/Users/fergbrain/Library/Python/2.7/lib/python/site-packages/ofxtools-0.5.1-py2.7.egg/ofxtools/Client.py", line 158, in request_profile
    profrq = PROFRQ(clientrouting='NONE', dtprofup=dtprofup)
  File "/Users/fergbrain/Library/Python/2.7/lib/python/site-packages/ofxtools-0.5.1-py2.7.egg/ofxtools/models/base.py", line 51, in __init__
    raise ValueError(msg)
ValueError: Can't set PROFRQ.dtprofup to 1990-01-01: '1990-01-01' is type '<type 'datetime.date'>'; can't convert to datetime
```

**Comment by csingley** [comment]
This bug was recently introduced by my attempt to fix [this other bug](https://github.com/csingley/ofxtools/issues/6) without sufficient testing... `datetime.date` instances are no longer accepted as input to Types.DateTime.unconvert().

This is fixed in [3d23f18](https://github.com/csingley/ofxtools/commit/3d23f18891e7fbd12c5c8fc6807d0e3ec4321f10) .

Thanks for reporting.

**Comment by fergbrain** [comment]
Fixed by 5723c26da4a1a1fbe0685c08e3da39349284b4f2
